pam/integration-tests/helpers_test.go

```diff
@@ -23,6 +24,12 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+var (
+	authdTestSessionTime     = time.Now()
+	authdArtifactsFolder     string
+	authdArtifactsFolderSync sync.Once
```

Comment:
The usage of the term "folder" for "directory" is a pet peeve of mine (unless it's in a graphical file manager) 😬 Do you mind using "dir" or "directory" instead?

```suggestion
	authdArtifactsDir     string
	authdArtifactsDirSync sync.Once
```